cert/cert.proto

```diff
@@ -35,6 +35,12 @@ message RawNebulaCertificateDetails {
     Curve curve = 100;
 }
 
+message RawNetwork {
```

Comment:
Because submessages are LEN types, it could make sense to use a string here, which would let us use netip.Addr.MarshalBinary.

I'm gonna ignore `bits` for now because it needs to be encoded somehow anyway

A submessage:
```
[LEN+id] [length] [ [hi id][hi varint] [lo id][lo varint] ] = 
min 1 + 1 + 2(1+1)  = 6, max 1 + 1 + 2(1+10) = 24
```

A `bytes` for address:
```
[LEN+id] [length] [16-byte address]  = 
1 + 1 + 16 = 18
```

A submessage can make sense if we think hi/lo of the v6 address are likely to be "small numbers", but if they're not, we could end up wasting bytes compared to a string-ier representation.

Reply:
I dont anticipate this to be the final types, `fixed64 Hi` and `uint64 Lo` is probably correct though. I agree avoiding the submessage is ideal as well. Avoiding protobuf would also be pretty cool :D